url_shortener_proj/app/Visitor.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Visitor extends Model
+{
+    protected $fillable = ['ip_address', 'url'];
+
+    public function urls()
+    {
+        return $this->belongsToMany('App\Url');
+    }
+}
```

Comment:
Please, don't introduce new features until we finish with features created in this branch since it will create confusion.

If you want to work on some other parts of this application, use another git branch.